source/_posts/Proof-of-concept-met-durable-functions.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Proof of concept met durable functions
+date: 2019-03-28 19:54:37
+tags:
+---
+
+
+Case omschrijving 
+---
+
+Voor een klant heb ik een Proof of Concept gemaakt om grote hoeveelheden xml te transformeren het gaat dan 3000 bestanden per keer met een totale grote van 18 gigabyte. Voor dit artikel kan ik wegens privacy niet de echte data gebruiken en heb ik een dataset gebruikt van https://www.kaggle.com/datasets het gaat hier om een dataset met landen en de wijnen welke uit het desbetreffende land komen. 
```

Comment:
Voor een klant heb ik een Proof of Concept gemaakt om grote hoeveelheden **XML bestanden** te transformeren. Het gaat om 3000 bestanden per keer met een totale grote van 18 gigabyte.